src/Components/SmartComponents/CVEs/CVEsTableToolbar.test.js

```diff
@@ -298,4 +242,70 @@ describe('CVEsTableToolbar', () => {
         });
     });
 
+    describe('CVEs without errate disabled', () => {
+        it('Should hide Advisory filter and and enable showing CVEs without errata', () => {
+            renderWithContext(
+                { cves: { meta: { cves_without_errata: false } } },
+                { canToggleCvesWithoutErrata: true }
+            );
+
+            fireEvent.click(screen.getByRole('button', { name: 'Conditional filter toggle' }));
```

Comment:
please prefer `await userEvent.click` over fireEvent. This also requires to be wrapped in async to avoid jest giving warning and make test less flaky.

Reply:
Done, you are right. userEvent is the one that is recommended by the RTL community.